src/types/CaseListQueryParams.ts

```diff
@@ -14,11 +14,12 @@ export type Urgency = "Urgent" | "Non-urgent" | undefined
 export type CaseListQueryParams = {
   orderBy?: string
   order?: QueryOrder
-  resultFilter?: Filter[]
+  resultFilter?: TriggerExceptionFilter[]
```

Comment:
I like that you renamed this, but I wonder if we could use a term that also used in the design, maybe:
```suggestion
  reason?: Reason[]
```

Reply:
Thanks Csaba! Have made these changes now.